### DISCUSSION TITLE
Migration from 0.45 to 0.46 / simple 3.0

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1623] [Help]
Hi there,
I am on 0.45 and really want the fixe for the images not working in posts, fixed in 0.46.
Also I am afraid it might break since I have a heavily modified SImple theme "2.8.3"
![image](https://github.com/user-attachments/assets/be352547-c318-4724-aa8f-9fd4c652a287)

Can I update and continue to use my current theme, without the page functions ?

**Comment by bobmitro** [comment]
Yes, it will work until you install the 3.0.0 theme update. Then, you will need to merge the overrided files.

P.S. Images work very well in Publii 0.45. The problem was only with images on pages, but it was fixed in the new build 16908. (https://github.com/GetPublii/Publii/issues/1621).
Always remember to create a site backup before updating.

**Reply by hubyhuby** [reply]
> Yes, it will work until you install the 3.0.0 theme update. Then, you will need to merge the overrided files.
> 
> P.S. Images work very well in Publii 0.45. The problem was only with images on pages, but it was fixed in the new build 16908. (#1621). Always remember to create a site backup before updating.

Here is the problem with upload of pictures (ubuntu 22.04) , with explaination in SOUND :

https://github.com/user-attachments/assets/58c2238d-5faf-4a38-b8da-c32a4e4e8e75

**Comment by bobmitro** [comment]
This thread has been closed because a similar topic is already being discussed #1610. Please continue the conversation there to keep the discussion organized and avoid fragmentation.
